website/docs/getting-started/quick-start.md

```diff
@@ -145,8 +150,7 @@ export default function App() {
         allFilms {
           films {
             id
-            title
-            director
+            ...Film_li
```

Comment:
On a quick read, the `_li` reads like a typo. Is it.. list? Is it supposed to be item? And then when I read the next component I guess it might be `ListItem`. Not a big deal, but `Film_listItem` etc. would likely be more readable.

Reply:
Was supposed to be "list item" like a `<li>` tag, but I agree it's confusing. It's repeated so many times that I wanted to keep it as short as possible. I went with `Film_item` which splits the difference, but is at least a word.